packages/react-composites/tests/browser/chat/app/server.ts

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
```

Comment:
Same here, use webpack dev server so that we can save the whole file of code here